sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,152 @@
+
+# Backwards Compatibility
+
+This change requires a hard fork of the Stacks blockchain
+
+# Activation
+
+## Voting Timeline
+
+Voting will begin at bitcoin block height 870,750, which occurs ~ Sunday, November 17th, 2024.
+Voting will conclude at bitcoin block height 872,750, which occurs ~ Sunday, December 1st, 2024.
+
+## Activation
+
+The SIP-029 STX emission schedule is designed to activate on Stacks 3.0 as defined in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md). Therefore, this SIP is only meaningful when SIP-021 activates.
+
+### Process of Activation
+
+Users can vote to approve this SIP with either their locked/stacked STX or with unlocked/liquid STX, or both. The SIP voting page can be found at [stx.eco](https://stx.eco). The criteria for the stacker and non-stacker voting is as follows.
+
+#### For Stackers:
+
+In order for this SIP to activate, the following criteria must be met by the set of Stacked STX:
+
+- At least 80 million Stacked STX must vote, with at least 80% (64 million) voting "yes".
+
+The voting addresses will be:
+
+| Vote | Bitcoin Address | Stacks Address | Msg | ASCII-encoded msg | Bitcoin script |
+| - | - | - | - | - | - |
+| yes      | `11111111111mdWK2VXcrA1e7in77Ux` | `SP00000000001WPAWSDEDMQ0B9J76GZNR3T` | `yes-sip-29` | `000000000000000000007965732d7369702d3239` | `OP_DUP` `OP_HASH160` `000000000000000000007965732d7369702d3239` `OP_EQUALVERIFY` `OP_CHECKSIG` |
+| no       | `111111111111ACW5wa4RwyeKgtEJz3` | `SP000000000006WVSDEDMQ0B9J76NCZPNZ`  | `no-sip-29`  | `00000000000000000000006e6f2d7369702d3239` | `OP_DUP` `OP_HASH160` `00000000000000000000006e6f2d7369702d3239` `OP_EQUALVERIFY` `OP_CHECKSIG` |
+
+The addresses have been generated as follows:
+
+- Encode `<message>` in ASCII, with 0-padding.
+- Use the resulting `<encoding>` in the Bitcoin script`OP_DUP` `OP_HASH160` `<encoding>` `OP_EQUALVERIFY` `OP_CHECKSIG`.
+- The Bitcoin address is the `base58check` of the hash of the Bitcoin script above.
+- The Stacks address is the `c32check-encoded` message.
+
+Stackers (pool and solo) vote by sending Stacks dust to the corresponding Stacks address from the account where their Stacks are locked.
+
+Solo stackers only can also vote by sending a bitcoin dust transaction (6000 sats) to the corresponding bitcoin address.
+
+#### For Non-Stackers:
+
+Users with liquid STX can vote on proposals directly at [stx.eco](https://stx.eco) using the Ecosystem DAO. Liquid STX is the user’s balance, less any STX they have locked in the PoX stacking protocol, at the block height at which the voting started (preventing the same STX from being transferred between accounts and used to effectively double vote). This is referred to generally as "snapshot" voting.
```

Comment:
@will-corcoran Has the proposal been deployed?

Reply:
@friedger can you clarify what you mean by 'deployed'? 

cc: @Hero-Gamer

Reply:
> @friedger can you clarify what you mean by 'deployed'?
> 
> cc: @Hero-Gamer

the question is if stx.eco has a section to vote on this SIP (it does not)

Reply:
Will this vote be simplified to just have non-stackers also send a dust amount to the STX addresses to vote?

Reply:
Hi @obycode - the intention is to have this vote run just like the SIP-028 vote (cc: @radicleart @Hero-Gamer )

@friedger - Mike is in the process of deplying the proposal.  The plan is to have all CAB votes completed by EOD Wed Nov 13 and then Mike will get the platform set up to host this vote.  The vote would then open on Sunday Nov 17.

Reply:
> the intention is to have this vote run just like the SIP-028 vote

Right, didn't that remove the DAO voting and just use the send to address voting for everyone (though it looks like the SIP text was not updated accordingly).

Reply:
Yes - your right @obycode the mention of Ecosystem Dao is confusing. It should say something like

"Users voting power is the sum of their locked and unlocked balance, calculated from the stacks transaction, at the height when voting began. User's can vote by visiting the [stx.eco](https://stx.eco) platform."

Reply:
thanks @radicleart I will updated the SIP to reflect your suggested revision later this afternoon.  @obycode are you good with this revised wording?

Reply:
Maybe simpler to merge the "For Stackers" and "Non Stackers" into one section to reflect that the same tx counts both their locked and unlocked balances.

Reply:
Agreed. Here's my suggestion for modifications (deleting the "For Non-Stackers:" section):

```md
### Process of Activation

 Users can vote to approve this SIP with either their locked/stacked STX or with unlocked/liquid STX, or both. The SIP voting page can be found at [stx.eco](https://stx.eco). The criteria for the stacker and non-stacker voting is as follows.

 #### To Vote:

 In order for this SIP to activate, the following criteria must be met:

 - At least 80 million stacked STX must vote, with at least 80% of all stacked STX committed by voting must be in favor of the proposal (vote "yes").
- At least 80% of all liquid STX committed by voting must be in favor of the proposal (vote "yes").

 The voting addresses will be:

 | Vote | Bitcoin Address | Stacks Address | Msg | ASCII-encoded msg | Bitcoin script |
 | - | - | - | - | - | - |
 | yes      | `11111111111mdWK2VXcrA1e7in77Ux` | `SP00000000001WPAWSDEDMQ0B9J76GZNR3T` | `yes-sip-29` | `000000000000000000007965732d7369702d3239` | `OP_DUP` `OP_HASH160` `000000000000000000007965732d7369702d3239` `OP_EQUALVERIFY` `OP_CHECKSIG` |


 | no       | `111111111111ACW5wa4RwyeKgtEJz3` | `SP000000000006WVSDEDMQ0B9J76NCZPNZ`  | `no-sip-29`  | `00000000000000000000006e6f2d7369702d3239` | `OP_DUP` `OP_HASH160` `00000000000000000000006e6f2d7369702d3239` `OP_EQUALVERIFY` `OP_CHECKSIG` |

 The addresses have been generated as follows:

 - Encode `<message>` in ASCII, with 0-padding.
 - Use the resulting `<encoding>` in the Bitcoin script`OP_DUP` `OP_HASH160` `<encoding>` `OP_EQUALVERIFY` `OP_CHECKSIG`.
 - The Bitcoin address is the `base58check` of the hash of the Bitcoin script above.
 - The Stacks address is the `c32check-encoded` message.

All STX holders vote by sending Stacks dust to the corresponding Stacks address from the account where their Stacks are held (stacked or liquid). Voting power is determined by a snapshot of the amount of STX (stacked and un-stacked) at the block height at which the voting started (preventing the same STX from being transferred between accounts and used to effectively double vote).

 Solo stackers only can also vote by sending a bitcoin dust transaction (6000 sats) to the corresponding bitcoin address.
```

Reply:
awesome.  thanks, @obycode and @radicleart

Reply:
Thanks!